swig/python2/tests/ConcurrentHelpers/SysrepoTester.py

```diff
@@ -74,14 +73,16 @@ def commitStep(self):
 
     def getItemsStep(self, xpath):
         self.session.get_items(xpath)
+        self.session.get_items(xpath)
 
     def getItemsStepExpectedCount(self, xpath, count):
         items = self.session.get_items(xpath)
-        self.tc.assertEqual(len(items), count)
+        self.tc.assertEqual(items.val_cnt(), count)
 
     def getItemsFailStep(self, xpath):
         with self.tc.assertRaisesRegexp(RuntimeError, ".* found"):
-            self.session.get_items(xpath)
+            vs = self.session.get_items(xpath)
+            if vs is None: raise (RuntimeError(".* found")) # likely a hack!
```

Comment:
Hi Antonio,

okey, let me know when I can start the review.

Regards,
Lukas